README.md

```diff
@@ -31,35 +31,35 @@ Permissions need Delegated Permissions to at least have "Enable sign-on and read
 Note: Seems like the terminology is still fluid, so follow the MS guidance (buwahaha) to set this up.
 
 The TenantInfo information can be a hash or class. It must provide client_id and client_secret.
-Optionally a domain_hint and tenant_id. For a simple single-tenant app, this could be:
+Optionally an aad_domain_hint and aad_tenant_id. For a simple single-tenant app, this could be:
 
 ```ruby
 use OmniAuth::Builder do
   provider :azure_oauth2,
     {
-      client_id: ENV['AZURE_CLIENT_ID'],
-      client_secret: ENV['AZURE_CLIENT_SECRET'],
-      tenant_id: ENV['AZURE_TENANT_ID']
+      aad_client_id: ENV['AZURE_CLIENT_ID'],
```

Comment:
By renaming the properties, it breaks existing usages. This would require a major version upgrade to comply w/ semver ... and is unneeded.  So, would need these changed back.